src/drivers/build/drivers-384.111.dockerfile

```diff
@@ -48,7 +48,36 @@ RUN apt-get -y update && \
         python-pip \
         realpath \
         gawk \
-        module-init-tools && \
+        module-init-tools \
```

Comment:
I really think we should have only one template file, and instantiated version specific file in build-pre.sh